backend/zerowaste/src/main/java/com/zerowaste/dtos/products/EditProductDTO.java

```diff
@@ -34,5 +35,7 @@ public record EditProductDTO(
     LocalDate expiresAt,
 
     @Min(0)
-    Integer stock
-) {} 
+    Integer stock,
+
+    List<Long> promotionsIds
```

Comment:
1. Eu recomendaria ser um `Set` em vez de uma `List`. Sendo assim, a própria estrutura de dados não permite que existam dois elementos iguais.
(editei a sugestão)

```suggestion
    Set<Long> promotionsIds
```